src/core/components/states/stateSkeletonCircular/stateSkeletonCircular.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { StateSkeletonCircular } from './stateSkeletonCircular';
+
+const meta: Meta<typeof StateSkeletonCircular> = {
+    title: 'Core/Components/States/StateSkeletonCircular',
+    component: StateSkeletonCircular,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/P0GeJKqILL7UXvaqu5Jj7V/v1.1.0?type=design&node-id=16106-38321&mode=design&t=8BGCESGcufn0gpI9-4',
+        },
+    },
+};
+
+type Story = StoryObj<typeof StateSkeletonCircular>;
+
+/**
+ * Default usage example of the StateSkeletonCircular component.
+ */
+export const Default: Story = {
+    args: {},
```

Comment:
Can we set a white background or a color with an higher contrast for these stories? It's hard to see the component with the current bg

Reply:
Defaulting to neutral-800